pkgs/top-level/beam-packages.nix

```diff
@@ -49,6 +49,10 @@ rec {
     # access for example elixir built with different version of Erlang, use
     # `beam.packages.erlangR19.elixir`.
     elixir = packages.erlang.elixir;
+    elixir-1_5 = packages.erlang.elixir-1_5;
```

Comment:
the naming convention for attributes is `elixir_1_5` or `elixir15` and since this is an rc release it should be suffixed to make that explicit.